News-Android-App/src/main/assets/web.css

```diff
@@ -1,3 +1,9 @@
+:root {
```

Comment:
later on you define this `root` section again programmatically. Isn't this a "duplicate" then? :)

Reply:
That's intentional - thinking is to have that in css as well as a fallback, in case the code isn't there, or whatever might go wrong. It works as desired, the programmatic definition overrides the css one.

Reply:
Okay, I was just thinking that. As it is defined later in the css it should override it, so it's all good 👍